app/views/spree/admin/products/_form.html.erb

```diff
@@ -32,7 +32,9 @@
         <%= f.field_container :status do %>
           <%= f.label :status, Spree.t(:status) %>
           <% options = ['draft', 'active', 'archived'].each_slice(1) %>
-          <% select_html = { class: 'select2 w-100', data: { action: 'change->product-edit#switchAvailabilityDatesFields','product-edit-target': 'status' } } %>
```

Comment:
@kgorazd we also need to make sure we're removing this attribute from permitted attributes so they cannot modify it on the backend